Symfony/CS/Config/Config.php

```diff
@@ -92,19 +89,7 @@ public function getFinder()
         return $this->finder;
     }
 
-    public function level($level)
-    {
-        $this->level = $level;
-
-        return $this;
-    }
-
-    public function getLevel()
-    {
-        return $this->level;
-    }
-
-    public function fixers($fixers)
+    public function fixers(array $fixers)
```

Comment:
could we add a PHPDoc on this what types are expected inside the array? I always get confused with the configuration between FixerInterface instances and names (strings).